templates/profiles/profile.html

```diff
@@ -38,6 +38,15 @@ <h2>{% blocktrans %}<span class="fn nickname">{{ user }}</span>{% endblocktrans
                     <span class="fn"><a href="https://gitlab.com/{{ local_profile.gitlab_url }}" target="_blank">{{ local_profile.gitlab_url }}</a></span>
                 </p>
             {% endif %}
+
+            {% if extra_fields%}
+                {% for field in extra_fields%}
+                    <p><b>{{ field.key }}:</b> {{ field.value }}</p>
```

Comment:
Do you see any issue with using? 

```suggestion
                    <p><b>{{ field.key }}:</b> {{ field.value|urlize }}</p>
```

`urlize` build our link with a `rel="nofollow"` attribute.

Reply:
The only potential issue would be TLDs that aren't supported by the tag which would impact .co, .co.uk, .dev, .io, etc. Not sure how big of a deal that would be :shrug:

Reply:
Per the [docs](https://docs.djangoproject.com/en/5.0/ref/templates/builtins/#urlize)

> It also supports domain-only links ending in one of the original top level domains (.com, .edu, .gov, .int, .mil, .net, and .org). For example, djangoproject.com gets converted.

Reply:
Leaving this mostly as a note for myself

> Check Trac to see if there are any tickets about increasing the coverage of the TLDs for urlize. [Trac](https://code.djangoproject.com/) is currently down (returning a 503 for me) so this will have to be done once it's back up. If there isn't a ticket to expand, write one up and/or post something in the Discord / Forum about the idea

Reply:
Interesting... TIL

hmm, I wonder if this is a potentially safe default to roll out or if it'd quickly annoy people.

Reply:
The alternative might be building out the link by hand but we might want to switch the value field to a https://docs.djangoproject.com/en/5.0/ref/models/fields/#urlfield if we go that route. 🤔

Reply:
> Interesting... TIL
> 
> hmm, I wonder if this is a potentially safe default to roll out or if it'd quickly annoy people.

I think for Mastodon users it could be an annoyance / confusing. Why does the github url render, but my Mastodon link doesn't (for me it wouldn't because i'm on https://mastodon.social/). 

If we set it to a `urlfield` then a proper url would have to be entered, right? Then a user couldn't put in other 'extra' information like preferred pronoun / gender or other non-url fields. 

That might be OK, but since we're starting from the Mastodon idea, there isn't any enforcement there (on Mastodon) for URLs. 

My idea of submitting a ticket to extend the TLDs supported would not fix the immediate need for this so, we might want to [take the code for the template tag](https://github.com/django/django/blob/ca5cd3e3e8e53f15e68ccd727ec8fe719cc48099/django/utils/html.py#L260) and extend it on our own with a custom version 🤔

Reply:
> Then a user couldn't put in other 'extra' information like preferred pronoun / gender or other non-url fields.

I hadn't thought of that. I think for pronouns we could just add them? 

I kind of thought this might be a URL link to the location, which an UrlField would be fine for. Then we could build our own <a href="" rel="nofollow"> links. 

Do you have as strong preference?

Reply:
What if we had one set of extra fields for URLs and used the `urlfield` and another set of extra fields for `text`? We might need to rename the models if that’s the direction to go.

Another option might be a flag to indicate that the field is a URL and that then becomes a `urlfield` or just has the same validation that `urlfield` would.

Reply:
My preference would be to complete the scope of "Add social media or custom links?" via the url field. We could do a new issue/follow up for custom attributes or fields if you wanted. 

We could also create more of a bio and expose that in the profile if you think that's more useful, too. (new issue though plz)

Reply:
Sounds good. I should be able to have an updated PR tomorrow 😄

Reply:
> Leaving this mostly as a note for myself
> 
> > Check Trac to see if there are any tickets about increasing the coverage of the TLDs for urlize. [Trac](https://code.djangoproject.com/) is currently down (returning a 503 for me) so this will have to be done once it's back up. If there isn't a ticket to expand, write one up and/or post something in the Discord / Forum about the idea

I found [this issue](https://code.djangoproject.com/ticket/16656) on Trac and it looks like a setting was already discussed and decided against (though it was 11 years ago). Also, when I added `urlize` tag it rendered the URL for mastodon.social so it could be that the docs need to be reviewed / updated